godel_calibration_config/yaml/example_caljob.yaml

```diff
@@ -0,0 +1,257 @@
+---
+reference_frame: world_frame
+scenes:
+-
+     scene_id: 0
+     trigger: ROS_ROBOT_JOINT_VALUES_ACTION_TRIGGER
+     trig_action_server: rosRobotSceneTrigger_joint_values
+     joint_values:
+        - -0.283292
+        - 0.169692
+        - 0
```

Comment:
Occasionally, applications that attempt to load a double from a ros parameter but get an integer instead may crash.  I there haven't been issues with the program that loads these parameters then ignore.